.github/workflows/wss-scan.yml

```diff
@@ -0,0 +1,46 @@
+name: WhiteSource Vulnerability Scan
+
+on:
+  schedule:
+    - cron: '30 3 * * *'
+  repository_dispatch:
+    types: [wss-scan]
+#  push:
+#    branches: [sreekarj_ws]
+
+jobs:
+  whitesource-vulnerability-scan:
+    name: WhiteSource Vulnerability Scan
+    runs-on: ubuntu-18.04
+    timeout-minutes: 80
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup Java
+        uses: actions/setup-java@v1
+        with:
+          java-version: 14
+
+      - name: Vulnerability Scan
+        env:
+          wss_apikey: ${{ secrets.WSS_API_KEY }}
+        run: |
+          cd standalone-tools
```

Comment:
We already have a bin folder, why have another folder called standalone-tools to further complicate the current structure?

Reply:
We had this discussed in the team meeting. The decision was made to create a separate folder and later move to its own individual repos when we have more tools.

Reply:
Ok that is fine, I will mark this as resolved.